open-company/open-company-web
# Email domain nux

Add the ability to configure an email domain so that members will automatically join your team during the NUX.

Testing steps:

While testing make sure it matches abstract: https://share.goabstract.com/653edc71-e66f-4603-8434-2196bdf68f7b

- sign up with a google account that has a domain (opencompany.com works with this test)
- when the team setup appears you should see `opencompany.com` or the domain in the input box for the new domains.
- add a company name and click continue
- after you are logged in open the settings
- [x] Do you see the domain in the settings?

- sign up with an account (email/google/slack) without a domain
- the input for an email domain should be empty
- add a domain in the input
- after logging in open the settings
- [x] Does the domain show up in settings?

- sign up with an account
- skip the domain and only put in a company name
- [x] are you logged in and nothing is added in settings?

- merge
